application/api/login.py

```diff
@@ -62,3 +67,53 @@ def post(self):
                 "data": {"user_id": user.user_id}
                 }
         return data, 200
+
+
+class ChangePassword(Resource):
+    """To change the user password"""
+
+    @token_required
+    def post(self, session):
+        """
+         To change the password for the user and updating new password in
+         database
+
+        Args:
+            session (object):By using this object we can get the user_id.
+
+        Returns:
+            Standard API Response with message(returns message saying
+            that password successfully changed) and http status code.
+
+        """
+        try:
+            post_db_detail_parser = reqparse.RequestParser(bundle_errors=True)
+            post_db_detail_parser.add_argument('old_password', required=True,
+                                               type=str,
+                                               help=APIMessages.PARSER_MESSAGE)
+            post_db_detail_parser.add_argument('new_password', required=True,
+                                               type=str,
+                                               help=APIMessages.PARSER_MESSAGE)
+
+            password_data = post_db_detail_parser.parse_args()
+            current_user_id = session.user_id
+            current_user_object = db.session.query(User).get(current_user_id)
```

Comment:
Instead of current_user_object = db.session.query(User).get(current_user_id)
you can use

current_user_object = session.user